src/game/client/components/console.cpp

```diff
@@ -151,6 +153,13 @@ void CGameConsole::CInstance::OnInput(IInput::CEvent Event)
 					char *pEntry = m_History.Allocate(m_Input.GetLength()+1);
 					mem_copy(pEntry, m_Input.GetString(), m_Input.GetLength()+1);
 				}
+				// if the user isn't authenticating or disconnected, show their upcoming commands in the scrollback
+				if (pConPrompt != "ENTER PASSWORD> " && pConPrompt != "NOT CONNECTED> ")
```

Comment:
You cannot compare strings like this in C++

Reply:
Fixed it